examples/annotated-skaffold.yaml

```diff
@@ -61,6 +61,15 @@ build:
     # bazel:
     #  target: //:skaffold_example.tar
 
+    # jibMaven is an artifacts built with Maven, typically (but not only) a Java project.
```

Comment:
nit: `artifacts` should be singular, or `is for artifacts`. Could I suggest the following?
```
# jibMaven builds containers using the Jib plugin for Maven
```

Do you see this file as being documentation?  Is it worth adding some detail about the `module` and `profile` items?